go.mod

```diff
@@ -40,10 +40,9 @@ require (
 	github.com/tklauser/go-sysconf v0.3.4 // indirect
 	github.com/vmihailenco/msgpack/v5 v5.1.0
 	github.com/yuin/gopher-lua v0.0.0-20191220021717-ab39c6098bdb
-	golang.org/x/tools v0.0.0-20200609124132-5359b67ffbdf // indirect
 	gopkg.in/yaml.v2 v2.3.0
 )
 
 replace github.com/docker/docker/internal/testutil => gotest.tools/v3 v3.0.0
 
-replace github.com/c-bata/go-prompt => github.com/binaek89/go-prompt v0.2.7-multiline-clearscreen
+replace github.com/c-bata/go-prompt => github.com/mrrvz/go-prompt v0.4.0
```

Comment:
AFAIU we change the dependency to fork with extended functionality. If so, then I would prefer this fork to be inside the "tarantool" organization.

Reply:
I think we need to involve @Totktonada

Reply:
Moved to tarantool org repo: https://github.com/tarantool/go-prompt